ui/app/templates/vault/cluster.hbs

```diff
@@ -4,13 +4,34 @@
 ~}}
 
 <Sidebar::Nav::Cluster />
-{{! Only show license banners for Enterprise }}
-{{#if this.activeCluster.version.isEnterprise}}
-  <LicenseBanners
-    @expiry={{this.activeCluster.licenseExpiry}}
-    @autoloaded={{eq this.activeCluster.licenseState "autoloaded"}}
-  />
-{{/if}}
+<div class="cluster-banners-wrapper">
```

Comment:
This class automatically adds margins to the child divs so that spacing is consistent